bokehjs/src/coffee/main.coffee

```diff
@@ -1,15 +1,10 @@
-Bokeh = {}
-Bokeh.require = require
-Bokeh.version = '0.9.2'
```

Comment:
This needs to be reverted, bokeh-scala depends on this being in this specific form.